Framework/PythonInterface/plugins/algorithms/VesuvioPeakPrediction.py

```diff
@@ -107,6 +107,23 @@ def PyExec(self):
 
                 vesuvio_params.addRow([temp, self._atomic_mass, self._debye_temp, kinetic, rms_momentum, rms_disp])
 
+        if self._model == "Classical":
```

Comment:
These seem like they'd be a good place to use a `match ... case` statement rather than `if`s, while we're editing this file.